modules/ROOT/pages/file/file-list.adoc

```diff
@@ -3,35 +3,83 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The List operation returns a List of Messages, where each message represents any file or folder found within the Directory Path (`directoryPath`).
 
-By default, the operation does not read or list files or folders within any sub-folders of `directoryPath`.
 
-To list files or folders within any sub-folders, you can set the `recursive` parameter to `true`.
+The List operation returns an Array of Messages, where each message represents any file
+or folder (and their attributes) found in the path pointed by the `directoryPath` parameter.
+This parameter is required and represents the *relative path* of the directory you want to list.
+This path is relative to the `working directory` defined in the configuration referenced by the
+`config-ref` parameter. To see an example of creating a configuration check
+xref:file/file-connector.adoc#connection_settings[this page]. If no configuration is referenced,
+the working directory defaults to the value of the `user.home` system property. If the system
+property is not set, the connector fails to initialize.
 
-This example lists all messages in a `directoryPath`, without listing the contents of any sub-folders. Note that the path can be absolute, or it can be relative to a Working Directory (`workingDir`) set in the File configuration (through `<file:config/>`).
+A simple example of the `list` operation:
 
 [source,xml,linenums]
 ----
-<flow name="list">
-  <file:list directoryPath="~/dropFolder" />
-  <foreach>
+<file:list config-ref="File_Config" directoryPath="relativePath" />
+----
+
+This will output an Array of Mule Messages, where the `payload` is the file's contents (or null if it is
```

Comment:
this is what I asked above, so it is like repeated, refactor the other paragraph